integrations/ktor/build.gradle.kts

```diff
@@ -82,5 +82,5 @@ mokkery {
 }
 
 datadogBuildConfig {
-    pomDescription = "The Ktor integration to use with the Datadog monitoring library for Kotlin Multiplatform."
+    pomDescription = "The Ktor 2 integration to use with the Datadog monitoring library for Kotlin Multiplatform."
```

Comment:
> [!NOTE]
> Probably it's worh it also rename `ktor` folder to `ktor2`?

Reply:
This will automatically change artifact name from `dd-sdk-kotlin-multiplatform-ktor` to `dd-sdk-kotlin-multiplatform-ktor2` and we want to keep backward compatibility, so that users on the `dd-sdk-kotlin-multiplatform-ktor` still continue to receive new versions.

It is possible to do additional configuration to explictly set artifact name to `dd-sdk-kotlin-multiplatform-ktor` despite the module name, but imo it is simpler just to keep old module name.